### PR TITLE
operator: fix missing model grid spacing s in viscoelastic equation.

### LINKED ISSUE #1169: Operator: missing model grid spacing "s" in viscoelastic equation.
Missing model grid spacing "s" in stress equation for viscoelastic example and ForwardOperator.

Actually:

```python

# Stress equations:
u_t = Eq(tau.forward,  model.damp *  (r.forward + tau +
                                      s * (l * t_ep / t_s * diag(div(v.forward)) +
                                           mu * t_es / t_s * (grad(v.forward) + grad(v.forward).T))))

```

After correction:

```python

# Stress equations:
u_t = Eq(tau.forward,  model.damp *  (s*r.forward + tau +
                                      s * (l * t_ep / t_s * diag(div(v.forward)) +
                                           mu * t_es / t_s * (grad(v.forward) + grad(v.forward).T))))

```
This correction must be performed in these files:

*  `/examples/seismic/tutorials/09_viscoelastic.ipynb`

* `/examples/seismic/viscoelastic/operators.py` (line 52)

### PR DESCRIPTION
**Pull request by nogueirapeterson**
Closes #1169

### CODE REVIEW

File: examples/seismic/viscoelastic/operators.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def ForwardOperator(model, geometry, space_order=4, save=False, **kwargs):
     u_v = Eq(v.forward, damp * (v + s*ro*div(tau)))
     symm_grad = grad(v.forward) + grad(v.forward).T
     # Stress equations:
-    u_t = Eq(tau.forward, damp * (r.forward + tau +
+    u_t = Eq(tau.forward, damp * (s*r.forward + tau +
```
Comment (mloubout) [inline examples/seismic/viscoelastic/operators.py]: You may need to change the norm to test against in `examples/seismic/viscoelastic/viscoelastic_example.py`

Comment (nogueirapeterson) [reply examples/seismic/viscoelastic/operators.py]: Ok. Sorry!
 I wiil do that.

Comment (nogueirapeterson) [reply examples/seismic/viscoelastic/operators.py]: I've already made the changes in `examples/seismic/viscoelastic/viscoelastic_example.py`

**Comment by rhodrin** [comment]
GPU tests ran manually and all passed. Merging.

**Pull request merged**